core/extension/gdextension_manager.cpp

```diff
@@ -387,6 +387,33 @@ bool GDExtensionManager::ensure_extensions_loaded(const HashSet<String> &p_exten
 	return needs_restart;
 }
 
+void GDExtensionManager::startup() {
```

Comment:
We're going to need to ensure that this also runs for GDExtensions that are loaded after Godot has already started. Maybe we could set a flag in this function if this has already run, and then when loading a GDExtension, if this flag is already set, then we know we'd need to run its startup callback right away?